.ci/travis/decoration-1.0.0.xsd

```diff
@@ -0,0 +1,466 @@
+<!-- copy from http://maven.apache.org/xsd/decoration-1.0.0.xsd ,
+     just to make build stable in offline mode.
+     Suorce was a bit formatted to pass LineLength and NewlineAtEndOfFile validation
```

Comment:
Suorce => Source

Reply:
sorry that I forgot to fix this in rush to fix major problem, now it is fixed in master